### PR TITLE
Add/login signup design

### LINKED ISSUE #1: User Story #1 - Login/Signup page design (UI)
1. Colors
1. Wireframe
1. Design of the welcome page

### PR DESCRIPTION
**Pull request by LeoNero**
Closes #1

### CODE REVIEW

**Review by haleyharelson — APPROVED** [review]

File: app/src/main/res/layout/activity_login.xml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        android:id="@+id/imageView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="8dp"
+        android:layout_marginLeft="8dp"
+        android:layout_marginTop="64dp"
+        android:layout_marginEnd="8dp"
+        android:layout_marginRight="8dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.497"
```
Comment (ghost) [inline app/src/main/res/layout/activity_login.xml:98]: Hard coded?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -2,12 +2,14 @@
...
     <string name="title_activity_home">HomeActivity</string>
 
-    <string name="username_signup">Username</string>
-    <string name="password_signup">Password</string>
-    <string name="email_signup">Email</string>
-    <string name="create_new_signup">create new</string>
-
-    <string name="prompt_email_signup">Email</string>
-    <string name="prompt_username_signup">Username</string>
-    <string name="prompt_password_signup">Password</string>
```
Comment (ghost) [inline app/src/main/res/values/strings.xml:12]: What did you replace this deletion with?

Comment (LeoNero) [reply app/src/main/res/values/strings.xml:12]: With the green lines below (10-12)

File: app/src/main/res/layout/activity_login.xml (hunk 1 above)
Comment (LeoNero) [reply app/src/main/res/layout/activity_login.xml:98]: Not sure... this was generated by the design stuff in android studio.

**Review by ghost — APPROVED** [review]

**Pull request merged**